gulpfile.js

```diff
@@ -70,6 +72,11 @@ function handleError() {
   });
 }
 
+// cleanup /dist in the initial build steps
+gulp.task('cleanup-dist', function (cb) {
+  del(['/dist/*'], cb);
+});
+
```

Comment:
I think this task is enough... I guess we don't really have to watch for changes and clean up `/dist` every time something is changed.  That seem like it will slow down build time and is only useful for local development.